### ISSUE TITLE
BreakoutFolder is incorrect while open PDF via MSEdge

### ISSUE THREAD

**Issue by Simba98** [sandboxie-plus/Sandboxie#2240]
### Describe what you noticed and did

1. Put the following to the [GlobalSettings]:
`BreakoutFolder=C:\Program Files (x86)\Microsoft\Edge`
2. Create a default sandbox.
3. Open an explorer in the sandbox and **double-click** a PDF file.
4. The msedge.exe will run inside the box but should be a breakout process.
![image](https://user-images.githubusercontent.com/7941967/190839707-7b8212b8-4687-4def-877c-26cbdbe5bc6e.png)

5. Replace the config in 1. to `BreakoutProcess=msedge.exe`
Works correctly.

### How often did you encounter it so far?

Everytime

### Affected program

MSEdge, explorer

### Download link

NA

### Where is the program located?

The program is installed only outside the sandbox.

### Expected behavior

Execute MSedge outside of sandbox

### What is your Windows edition and version?

Windows 10 Enterprise 19043.2006

### In which Windows account you have this problem?

A local or Microsoft account without special changes.

### Please mention any installed security software

NA, but MD

### What version of Sandboxie are you running?

v1.3.2 64Bit

### Is it a new installation of Sandboxie?

I have been using the same version for some time.

### Is it a regression?

NA

### In which sandbox type you have this problem?

In a Standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on an empty sandbox?

I can confirm it also on an empty sandbox.

### Did you previously enable some security policy settings outside Sandboxie?

NA

### Sandboxie.ini configuration

```shell
[GlobalSettings]
FileRootPath=\??\%SystemDrive%\Sandbox\%USER%\%SANDBOX%
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
Template=WindowsRasMan
Template=WindowsLive
Template=OfficeLicensing
Template=OfficeClickToRun
Template=Edge_Win11Fix
Template=7zipShellEx
BreakoutFolder=C:\Program Files (x86)\Microsoft\Edge
#BreakoutProcess=msedge.exe
BreakoutFolder=C:\Program Files\Microsoft VS Code
NetworkEnableWFP=y

[Test]
Enabled=y
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#02f6f6,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=9
AutoRecover=y
UseSecurityMode=n
UsePrivacyMode=n
```

**Comment by Simba98** [comment]
`BreakoutFolder=C:\Program Files (x86)\Microsoft\Edge\Application`
Works.
Need specify sub-folder

**Comment by Luro223** [comment]
@Simba98 if it works then close the issue

**Comment by Simba98** [comment]
> @Simba98 if it works then close the issue

Still needs documents, how can I specify the folder Recursion?

**Comment by Luro223** [comment]
> Still needs documents, how can I specify the folder Recursion?
let me check

**Comment by Luro223** [comment]
@Simba98 try `BreakoutFolder=C:\Program Files (x86)\Microsoft\Edge\*`

**Comment by Simba98** [comment]
> @Simba98 try `BreakoutFolder=C:\Program Files (x86)\Microsoft\Edge\*`

Still incorrect, this one won't include C:\Program Files (x86)\Microsoft\Edge\abc.exe
Although there is nothing for MS Edge, but, for example, for VS Code:
`BreakoutFolder=C:\Program Files\Microsoft VS Code\*` won't work, because the C:\Program Files\Microsoft VS Code\Code.exe do not match.

**Comment by Simba98** [comment]
And also, you may try `BreakoutFolder=C:\Program Files (x86)\*`

**Comment by Luro223** [comment]
@Simba98 it's because you're trying to launch microsoft edge by opening the associated file if you want to open by associated file you must use `BreakoutProcess`

**Comment by Luro223** [comment]
If I try to open executable file it will work for me if I put * but since I am not a supporter, the error message shows that this feature is for supporters only

**Comment by Simba98** [comment]
> @Simba98 it's because you're trying to launch microsoft edge by opening the associated file if you want to open by associated file you must use `BreakoutProcess`
No, the above testing can be reproduced by double click the exe file directly.
After adding, `BreakoutFolder=C:\Program Files (x86)\*`
![image](https://user-images.githubusercontent.com/7941967/190855052-ca63be8c-8409-4323-aecb-478c968ab655.png)

**Comment by Luro223** [comment]
@Simba98 because BreakoutFolder works by calling the file directly instead of indirectly through file associations

**Comment by Luro223** [comment]
@Simba98 Always glad to help. You can close issue now
Have a great time

**Comment by Simba98** [comment]
> @Simba98 Always glad to help. You can close issue now Have a great time

Have a nice day~
